bin/build.sh

```diff
@@ -10,13 +10,21 @@ find 'documents' -name '*.md' | while read file; do
 
   html_name="${file%.md}.html"
   pdf_name="${file%.md}.pdf"
+  stylesheet="../../../assets/style.css"
+  
+  file_headline=$(head -n 1 "$file")
+  document_title=$(echo "$file_headline" | sed -E "s/#+\s*//")
 
   # deliberately NOT using the pipe | operator here because wkhtml tries to resolve image links
   # based on the location of its input file. If piping is used, wkhtml assumes /tmp as location
   # and links like "./dummy-image.png" become "/tmp/dummy-image.png" instead of "edudoc/$project/dummy-image.png"
   sed -r "s#wca\{([^}]*)\}#$wca_url\1#g" "$file" | # Replace wca{...} with absolute WCA URL.
-  pandoc -o "$html_name" # Markdown -> HTML
-  wkhtmltopdf --encoding 'utf-8' --user-style-sheet 'assets/style.css' -T 15mm -B 15mm -R 15mm -L 15mm --quiet "$html_name" "$pdf_name" # HTML -> PDF
+  pandoc -s --from markdown --to html5 --css="$stylesheet"  --metadata pagetitle="$document_title" "$file" -o "$html_name" # Markdown -> HTML
+  wkhtmltopdf --encoding 'utf-8' -T 15mm -B 15mm -R 15mm -L 15mm --quiet "$html_name" "$pdf_name" # HTML -> PDF
+  
+  #ORIGINAL
```

Comment:
Don't check this in to the repository code please. We can always dig up earlier states of the command through git history.

Reply:
That's my bad, I left it in by accident